README.md

```diff
@@ -17,3 +17,8 @@ To disable rule on next line comment '// eslint-disable-next-line' followed by t
 See .eslintrc.jc for rules.
 To fix all lint errors 'npm run lint -- --fix'
 https://eslint.org/docs/latest/rules/
+
+## Prettier
+
+To find errors 'Npm run prettier'
```

Comment:
when giving examples in markdown try to use the tripple backtick syntax:

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

Reply:
@SWhyteAnswer looks like you added 3 apostrophes ' instead of 3 backticks `